packages/cli/src/util/paths.ts

```diff
@@ -0,0 +1,168 @@
+import { generate } from '@stoplight/prism-http/src/mocker/generator/HttpParamGenerator';
+import { serializeWithDeepObjectStyle } from '@stoplight/prism-http/src/mocker/serializer/style/deepObject';
+import {
```

Comment:
This is likely the reason why the harness tests are failing (well at least we know the test harness is working)

Basically, while in _test_ files it's ok to import from the source files directly, this will not work in production. The reason is that after the TypeScript compilation this becomes `dist` — and such directory does not exist anymore.

If you need stuff from the http-package, just export it explicitly from the `index.ts` file and grab it from there

@StefanDywersant

Reply:
@XVincentX Http's `index.ts` looks pretty concise. Only `createInstance` is actually exported. That is another vote on moving serializers to CLI which makes that solution win the election. WDYT?

Reply:
Forget ^. I exported it from http package.